.github/workflows/docs/pr-batching.md

```diff
@@ -49,7 +52,7 @@ jobs:
     name: Set automerge on opened PRs targeting the tracking branch
     permissions:
       contents: write
```

Comment:
I've removed this because it looks like the (public) RiffRaff workflow is failing on a permissions error, although I haven't been able to test in RiffRaff so I'm not 100% confident that it's erroring because it lacks this permission.